resources/translations/fi.edn

```diff
@@ -183,6 +185,8 @@
              :show "Näytä"
              :show-less "Näytä vähemmän"
              :show-more "Näytä lisää"}
+  :change-form {:change-form-intro "Olet muuttamassa seuraavien aineistojen lomaketta. Muutokset tehdään yksi kerrallaan siten, että vanha aineisto lopetetaan ja uuteen aktivoitavaan kopioon laitetaan valitsemasi lomake."
```

Comment:
Also, how about "uuteen aktivoitavaan kopioon laitetaan valitsemasi lomake" -> "uusi aktivoitava kopio käyttää valitsemaasi lomaketta"? I think using "laittaa" in the above meaning still has a somewhat casual style, so it's not very formal language.